pkg/controller/masteruserrecord/sync_test.go

```diff
@@ -86,16 +86,9 @@ func setupSynchronizerItems() (toolchainv1alpha1.MasterUserRecord, toolchainv1al
 	base := toolchainv1alpha1.UserAccountSpecBase{
 		NSLimit: "limit",
 		NSTemplateSet: toolchainv1alpha1.NSTemplateSetSpec{
-			TierName: "basic",
-			ClusterResources: &toolchainv1alpha1.NSTemplateSetClusterResources{
-				Revision: "654321a",
-			},
-			Namespaces: []toolchainv1alpha1.NSTemplateSetNamespace{
-				{
-					Revision: "123456a",
-					Type:     "code",
-				},
-			},
+			TierName:         "basic",
+			ClusterResources: &toolchainv1alpha1.NSTemplateSetClusterResources{},
+			Namespaces:       []toolchainv1alpha1.NSTemplateSetNamespace{},
```

Comment:
This should contain a namespace with `TemplateRef` if I'm not mistaken

Reply:
good catch! added here: 3e316e28db125cd9b1d08f0e366e910b9b54fc69